### PR TITLE
Pulling in changes from forked version

### PR DESCRIPTION
**Pull request by lowderchris**
Pulling in some changes from the development forked version.

### CODE REVIEW

**Comment by lowderchris** [comment]
@d-lamb - Does this look alright to merge in?

File: README [hunk 1]
```diff
@@ -1,44 +1,25 @@
...
+The authors gratefully acknowledge support from NASA's Living With A Star and Solar/Heliospheric Physics programs, and from the Southwest Research Institute.
+
+All source files are copyright (C) 2004-2023 by the authors unless otherwise marked.  You may modify and/or make copies of this software under the terms of the Gnu Public License, version 2 (GPL v2).  You should have received a copy of the GPL v2 with this software; it is present in the file "COPYING".  If you do not have a copy you may obtain one online from "http://www.gnu.org/licenses/gpl-2.0.html" or by writing to the Free Software Foundation, 59 Temple Place, Suite 330, Boston, MA 0211-1307 USA.
 
 THIS SOFTWARE COMES WITH NO WARRANTY OF ANY KIND.
 
 Authors:
-	Craig DeForest	 	deforest@boulder.swri.edu
-	Charles Kankelborg 	kankel@solar.physics.montana.edu
+	Craig DeForest		deforest@boulder.swri.edu
```
Comment (d-lamb) [inline README:19]: Maybe delete all these email addresses? Replace them with the institutions they had when they were contributing.  Laurel was a CU grad student, Alisdair was a SwRI employee.
Something like:

> Laurel Rachmeler (then at University of Colorado Boulder & SwRI)
> Alisdair Davey (then at SwRI)

File: README [hunk 2]
```diff
@@ -50,33 +31,17 @@ Contributors (gratefully acknowledged):
...
-    Perl will do the Right Thing.
-
-After installation, some autoloading routines will be installed for
-you.  See the docs for PDL::AutoLoader
-(http://pdl.perl.org/PDLdocs/PDL/AutoLoader.html) for a description of
-this handy feature of PDL.  Instructions come at the end of the
-installation script.
+    Where the Perl front-end module should be installed. This location should be in Perl's @INC list. Try doing ' perl -e "print join(qq|\n|,@INC,q||);" ' at the command line to see the contents of this list. If $PL_PREFIX/lib/perl5 is not included in @INC, then Perl will not be able to find the Flux module after installation. You can add $PL_PREFIX/lib/perl5 to your PERL5LIB environment variable to solve that problem.  Or you can just leave PL_PREFIX undefined and Perl will do the Right Thing.
+
+After installation, some autoloading routines will be installed for you.  See the docs for PDL::AutoLoader (http://pdl.perl.org/PDLdocs/PDL/AutoLoader.html) for a description of this handy feature of PDL.  Instructions come at the end of the installation script.
```
Comment (d-lamb) [inline README:44]: New link: https://metacpan.org/pod/PDL::AutoLoader

File: README [hunk 3]
```diff
@@ -89,25 +54,25 @@ installation script.
...
 Note the 'sudo' on lines 2 and [if required based on PL_PREFIX] 4
 
-$ export FL_PREFIX PL_PREFIX
-$ make libbuild
-$ sudo make libinstall
-$ make pdlbuild
-$ make pdltest
-$ [sudo] make pdlinstall
-$ export -n FL_PREFIX PL_PREFIX
+% export FL_PREFIX PL_PREFIX
```
Comment (d-lamb) [inline README:57]: "%" means "as root", is that what you want?

File: pdl/PDL/calculate_flow_in_fluxon.pdl [hunk 4]
```diff
@@ -16,114 +16,135 @@ sub calculate_flow_in_fluxon {
...
-
+    # Calculate array of positional radius and areas along the fluxon
+    # Work along the correction direction depending on which end is open
     if($en_open) {
-	our $r = ($me->dump_vecs->(0:2)**2)->sumover->sqrt * $opt{r0};
-	our $A = pdl(map {$_->{A}} ($me->vertices)) * $opt{r0} * $opt{r0};
+        our $r = ($me->dump_vecs->(0:2)**2)->sumover->sqrt * $opt{r0};
+        our $A = pdl(map {$_->{A}} ($me->vertices)) * $opt{r0} * $opt{r0};
     } else {
-	our $r = ($me->dump_vecs->(0:2,-1:0:-1)**2)->sumover->sqrt * $opt->{r0};
```
Comment (d-lamb) [inline pdl/PDL/calculate_flow_in_fluxon.pdl:45]: good catch!

File: doc/flux.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+System requirements
+-------------------
+
+### Software environment
+
+FLUX needs:
+
+-   a POSIX system under which to run. It has been tested under Fedora Linux, Ubuntu Linux, and MacOS X. It is thought to work under Cygwin (but why bother?)
+
+-   Perl 5.8.x or 5.10.x (standard on most POSIX systems these days)
```
Comment (d-lamb) [inline doc/flux.md:41]: I would say Perl 5.10 or higher. I believe that is the baseline for PDL.

File: doc/flux.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+-   PDL 2.4.x (standard on most Linuxes and available through Fink; source is available from [<http://pdl.perl.org>](http://pdl.perl.org)
```
Comment (d-lamb) [inline doc/flux.md:43]: I would say whatever PDL version introduced the most-recent API change that FLUX is compatible with.

File: doc/flux.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+In a direct comparison with a more conventional MHD model (ARMS) in early 2008, FLUX obtained similar answers in its appropriate computational domain, but used under 1% of the computing power as ARMS.
+
+System requirements
+-------------------
+
+### Software environment
+
+FLUX needs:
+
+-   a POSIX system under which to run. It has been tested under Fedora Linux, Ubuntu Linux, and MacOS X. It is thought to work under Cygwin (but why bother?)
```
Comment (d-lamb) [inline doc/flux.md:39]: I would take out the Cygwin obnoxious comment, and perhaps the whole Cygwin sentence.  It has been tested under (whatever).

**Review by d-lamb — CHANGES_REQUESTED** [review]
I can't rebase this due to conflicts, but there are enough changes here that a merge commit is totally fine.  I made a few comments.  If you want to address those then I think merging it in is OK, as long as it passes `make test`.

**Pull request merged**